src/pages/product-listing/ProductListing.jsx

```diff
@@ -1,15 +1,60 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Navbar from '../../components/navbar/Navbar';
 import Sidebar from '../../components/sidebar/Sidebar';
 import classes from './ProductListing.module.css';
-import img1 from '../../assets/celtics.jpeg';
-import img2 from '../../assets/lakers.jpeg';
-import img3 from '../../assets/76ers.jpeg';
-import img4 from '../../assets/clippers1.jpeg';
-import img5 from '../../assets/grizzlies.jpeg';
 import ProductCard from '../../components/card/ProductCard';
+import { useProducts } from '../../context/product-listing/product-context';
```

Comment:
You can have a file named as index.js from where you can have multiple exports then you won't have to write such big file paths.